app/src/main/java/org/ieeeguc/ieeeguc/models/User.java

```diff
@@ -95,6 +96,47 @@ public String getCommitteeName() {
         return committeeName;
     }
 
+    public static void getUser(String token, int id, final HTTPResponse httpResponse){

```

Comment:
move the comments above the method signature 
BTW you can generate the comment by adding `/**` above the method signature and then press `CTRL + space`